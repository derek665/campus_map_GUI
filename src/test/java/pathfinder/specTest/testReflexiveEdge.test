# test correct output when start node and end node are the same, and it has a reflexive edge

CreateGraph testReflexive

AddNode testReflexive A

AddEdge testReflexive A A 100.0

FindPath testReflexive A A