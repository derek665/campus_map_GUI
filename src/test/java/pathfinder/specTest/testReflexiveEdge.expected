# test correct output when start node and end node are the same, and it has a reflexive edge

created graph testReflexive

added node A to testReflexive

added edge 100.000 from A to A in testReflexive

path from A to A:
total cost: 0.000